docs/ruleset.rst

```diff
@@ -43,6 +43,14 @@ Left-hand sides are currently a few special forms which select nodes to be fed t
 
 .. autofunction:: and(typeCall[, typeCall, ...])
 
+.. function :: when(predicate)
+
+    Further specify types of node you'd like to select. Can be chained with type() or dom().
```

Comment:
Also, you can do the fancy formatting on type() and dom() so they link to the docs for those routines. I believe it's `:func:type()`, probably with a bit more to disambiguate it from RHS's type().